drivers/frequency/adf5355/adf5355.h

```diff
@@ -75,18 +75,18 @@
 /* REG5 Bit Definitions */
 #define ADF5355_REG5_DEFAULT			        0x00800025
 
-/* REG6 Bit Definitions */ //CHANGE // INIT for > 75MHz 
-#define ADF5355_REG6_OUTPUT_PWR(x)		        (((x) & 0x3) << 4)
-#define ADF5355_REG6_RF_OUT_EN(x)		        	((x) << 6)
+/* REG6 Bit Definitions */
 #define ADF4355_REG6_OUTPUTB_PWR(x)		        (((x) & 0x3) << 7)
 #define ADF4355_REG6_RF_OUTB_EN(x)		        ((x) << 9)
+#define ADF4356_REG6_RF_OUTB_SEL(x)                ((x) << 25)
```

Comment:
this looks like an addition. isn't this commit suppose to fix only format issues?